lib/releases.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 
-# shellcheck disable=SC1091
-source lib/common.sh
```

Comment:
no, this is required! some variables are defined below, and should come from lib/common.sh, like CAPM3_VERSION.

Reply:
Maybe a dump question, but how CAPM3_VERSION variable was sourced from lib/common.sh even when it was not there before #498 merged?

Reply:
I think the problem is here : https://github.com/metal3-io/metal3-dev-env/blob/8857751d750e33f210b114f05f929ca24775ad64/scripts/run.sh#L17
So, it is sourcing releases.sh from "${METAL3_DIR}/lib/" and but when it comes to sourcing common.sh from within the releases.sh it tries to find it from lib/ not "${METAL3_DIR}/lib/"

Reply:
And @maelk actually removing it should be fine, because wherever we call releases.sh we always first source the common.sh. Of course, this is not the best option, because we always have to remember to execute common.sh before we execute releases.sh, but this answers your question above @furkatgofurov7

Reply:
@maelk wdyt?

Reply:
yes, you can remove it